content/source/docs/enterprise/system-overview/architecture.html.md

```diff
@@ -5,24 +5,201 @@ page_title: "Architecture Summary - System Overview - Terraform Enterprise"
 
```

Comment:
Could you add a metadata description to the top of this page? 

description: |- 
  Learn about the functional layers of the architecture, how they interact, and details about the corresponding Docker containers.